docker-compose.yml

```diff
@@ -1,29 +1,44 @@
-version: "3"
+version: '3'
 
 services:
   server:
     image: node:12.9.1
     command: >
-      yarn install
-      yarn dev
+      /bin/sh -c '
```

Comment:
We should update Instructions for Dev (Install Postgress / Run postgress with docker / Run server ...)